### PR TITLE
Show / Ask HN implementation

### LINKED ISSUE #4: [feature] Adding Ask HN / Show HN

### PR DESCRIPTION
**Pull request by lambdatastic**
- Added server-side cache for show / ask HN feeds
- Created controllers and views for show / ask HN in front end
- Updated Gruntfile to include all new files

Closes #4

### CODE REVIEW

File: public/app/ask/ask.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div ng-controller="AskController">
+  <div class="userDisplay story" ng-repeat="story in stories | limitTo: index">
+    <div ng-include="'app/partials/story.html'" class="story-holder"></div>
+  </div>
+
+  <div class="more-button" ng-show="index < stories.length && stories.length > 0" ng-click="index = index + perPage;">more</div>
```
Comment (kf6nux) [inline public/app/ask/ask.html:6]: Did we want to use in-view instead of ng-click here as well?

**Comment by alicekamada** [comment]
Yeah, but I didn't want to clobber your commit and cause a merge conflict.

File: public/app/ask/ask.html (hunk 1 above)
Comment (lambdatastic) [inline public/app/ask/ask.html:6]: Yeah, but I didn't want to clobber it and cause a conflict. Gonna do a quick branch after this to fix it. Alternatively, I can rebase it now and just make the changes.

**Comment by kf6nux** [comment]
:+1:

**Pull request merged**